cli/commands/create/create.go

```diff
@@ -3,59 +3,145 @@ package create
 import (
 	"bytes"
 	"encoding/json"
+	"encoding/xml"
 	"errors"
 	"fmt"
 	"io"
 	"io/ioutil"
+	"net/http"
+	"net/url"
 	"os"
+	"path/filepath"
 	"regexp"
+	"strings"
 
 	"github.com/ghodss/yaml"
 	"github.com/sensu/sensu-go/cli"
 	"github.com/sensu/sensu-go/cli/client"
-	"github.com/sensu/sensu-go/cli/commands/helpers"
 	"github.com/sensu/sensu-go/types"
 	"github.com/spf13/cobra"
 )
 
 // CreateCommand creates generic Sensu resources.
 func CreateCommand(cli *cli.SensuCli) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "create [-f FILE]",
-		Short: "create new resources from file or STDIN",
+		Use:   "create [-r] [[-f URL] ... ]",
```

Comment:
I could see URL be a little confusing, but the short description does describe it.